src/main/java/core/basesyntax/DataConverter.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
```

Comment:
The `DataConverter` interface should be placed in the `service` package according to the checklist item about keeping logic in separate packages.